packages/cmr-client/src/getConcept.ts

```diff
@@ -3,35 +3,53 @@ import Logger from '@cumulus/logger';
 
 const log = new Logger({ sender: 'cmr-client' });
 
-/**
- * Get the CMR JSON metadata from the cmrLink
- *
- * @param {string} conceptLink - link to concept in CMR
- * @param {Object} headers - the CMR headers
- * @returns {Object} - metadata as a JS object, null if not
- * found
- */
-async function getConceptMetadata(
+async function getGranuleConceptMetadata(
   conceptLink: string,
-  headers: Headers
+  headers: Headers,
+  type: string
```

Comment:
If we're only going to accept two values for the type, we should make a type for `ConceptType` or something and enforce it as the argument type

Reply:
Right, it could be an enum.

Reply:
I was even just thinking:

```
Type ConceptType = 'granuleSearch' | 'granuleConceptSearch';
```

and then:

```suggestion
  type: ConceptType
```

Reply:
https://github.com/nasa/cumulus/pull/2014/commits/301666f42c859edac6ad6dde10c0ec25effb7446 👍